### ISSUE TITLE
Regarding: http://holmen.labscodium.net

### ISSUE THREAD

**Issue by KjellPingbull** [innocode-digital/labscodium#1]
I do have a question regarding a quiz-site for a customer: 
http://holmen.labscodium.net

Earlier we moved all our sites from Labscodium to pingbull01.se-server.
Customer is still pointing their quiz to labscodium.net.
Labscodium will soon shut down their servers, so this site needs to be up and running on av Pingbull stage and liver server.

Question is: 
1. Can you set up a stage (I believe this allready exists) and a live version of this site on Pingbull-servers?
2. Can Apriorit/Labscodium accès this site on Pingbull-servers to make updates and changes?

Please let me know if you need further information.

**Comment by KjellPingbull** [comment]
Hello @shyshkov! Have you had a chance to look into this errand?

**Comment by kuliebiakin** [comment]
holmen.labscodium.net - is already on Pingbull's AWS servers, we can just create a new domain, add A record to needed IP and setup config with new domain on server, if you mean that labscodium.net will not work soon, do we need to do it?

Since the site as I understand is not "live" yet, it's hard to say for me is holmen.labscodium.net a prod or stage ... Maybe you know?
We can use e.g. it as prod and create stage env.
I think guys from Apriorit (Webcodium?) has an access to these servers, since they gave us needed keys as I know. If not, please tell us and we will thinking how to do it.
If guys from Apriorit will work with this project and we need a stage then we need to create it on Pingbull's AWS server (not Innocode) since we will not able to give them access to Innocode servers.
But actually I think that they can work currently with holmen.labscodium.net (I can setup new domain, if needed, as described above) and when site will launch we will duplicate it to stage environment.

**Comment by KjellPingbull** [comment]
Hola. Well, as this is all on our servers its all good. We´ll update the domain name later on.

Regarding, the users from Apriorit and Webcodium, we are going to remove them as they should be doing further work on our servers. Is this something we can fix our selves, or should some of you guys with the right skills do this?

**Comment by kuliebiakin** [comment]
Should we remove access for them now or after they will finish some work?

**Comment by KjellPingbull** [comment]
Hi there. You can remove them now. They are not doing any work there now.

**Comment by kuliebiakin** [comment]
Hi @innocode-devops 
Can you please take a look at this? 
Probably their ssh keys are removed from servers (I just checked Live_2017 and see only ours), but I think they still able to login with keys which were sent to us some time ago from their DevOps guy.
Also I see Apriorit_.... IPs in security groups, should we remove it?

**Comment by KjellPingbull** [comment]
Yes, you can remove all access from them as we are not doing any more work with them.
